### ISSUE TITLE
Mac release is broken (does not run)

### ISSUE THREAD

**Issue by HomebrewHomunculus** [HomebrewHomunculus/BrogueLite#13]
Originally reported by IAmMarwood on [reddit](https://old.reddit.com/r/roguelikes/comments/n9fg3k/new_brogue_lite_its_brogue_but_even_easier_to_get/gxs3lh9/):

> One small problem though, I can't run the Mac version. It's not because it's unsigned, it's because it's broken. I get the no entry sign over the icon like when an app is the wrong architecture.

**Comment by SpudGunMan** [comment]
I did just download and build manually on 11.3.3 with success FYI

**Comment by HomebrewHomunculus** [comment]
Yep, I just confirmed this as well: compiling from source works, but it's the release doesn't, so it's the Github Actions build that is broken

**Comment by HomebrewHomunculus** [comment]
[Hoooo boy...](https://github.com/HomebrewHomunculus/BrogueLite/commit/6e02edd71383a4705405fb98a833df79e9907d5e#diff-1c59cf5670f2a12e5705c46fd8906c4809f5ba5d4a04a5367f69863b5c11d1c8R18) we'll see if this works...

[hotfix2-macos-x86_64.zip](https://github.com/HomebrewHomunculus/BrogueLite/files/6487945/hotfix2-macos-x86_64.zip)

**Comment by HomebrewHomunculus** [comment]
Ok, that seems to work - made a new hotfix release:

https://github.com/HomebrewHomunculus/BrogueLite/releases/tag/v1.9.3-hotfix-2
